client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,72 @@
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import './Navbar.css';
+
+const Navbar = () => {
+  const navigate = useNavigate();
+
+  const logout = () => {
+    // log user out
+    localStorage.clear();
+    navigate('/dashboard');
+  };
+
+  const user = useSelector((state) => state.user);
+  if (user) console.log(user);
+
+  let userInfo;
+  if (user) {
+    userInfo = (
```

Comment:
I haven't set up the navbar yet on my branch so I will utilize this for reference to build it to fit with your back end stuff.